lightning/src/ln/channelmanager.rs

```diff
@@ -2833,20 +2833,20 @@ impl<'a> PersistenceNotifierGuard<'a, fn() -> NotifyOption> {
 	/// isn't ideal.
 	fn notify_on_drop<C: AChannelManager>(
 		cm: &'a C,
-	) -> PersistenceNotifierGuard<'a, impl FnMut() -> NotifyOption> {
+	) -> PersistenceNotifierGuard<'a, impl FnOnce() -> NotifyOption> {
```

Comment:
So this means that only single-use closure may be provided now. And it seems we are lucky that nowhere a closure was reused, so no other refactoring needed? (questions for me to understand this better)

Reply:
No reuse inside `PersistenceNotifierGuard` obviously, but I wondered whether there could be a reason to reuse a closure higher up the call stack? (still bg q this, and learning rust